build/build.sh

```diff
@@ -28,11 +32,14 @@ pandoc --verbose \
 
 # Create PDF outpout
 echo "Exporting PDF manuscript"
+cd output
```

Comment:
Okay after some research, this makes me think we should be building from the `output` directory if we're going to be using relative links. If we move track `output/images` and put images there, then relative links should work for PDF and HTML exports. Downside is that figures won't show up in text editor. Your original copy directory method would solve this, but I don't like the duplication... the repo structure is already complicated.

What do you think about only supporting images by URL for now? One downside is you need a network connection... but you already need that to build. Also harder to have images hosted in private repos. I'm on the fence here... @agitter and @slochower what are your thoughts?